### PR TITLE
rust: add badges for crates.io

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: bindings/rust/Cargo.toml [hunk 1]
```diff
@@ -15,3 +15,8 @@ edition = "2018"
 [build-dependencies]
 bindgen = "0.54.0"
 cmake = "0.1"
+
+[badges]
```
Comment (axic) [inline bindings/rust/Cargo.toml:19]: See https://doc.rust-lang.org/cargo/reference/manifest.html#the-badges-section

File: bindings/rust/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Fizzy
```
Comment (axic) [inline bindings/rust/README.md:1]: This is the readme which will be displayed on crates.io

File: bindings/rust/Cargo.toml [hunk 3]
```diff
@@ -15,3 +15,8 @@ edition = "2018"
...
+circle-ci = { repository = "https://github.com/wasmx/fizzy" }
+codecov = { repository = "https://github.com/wasmx/fizzy" }
```
Comment (axic) [inline bindings/rust/Cargo.toml:21]: Not sure if this should be:
```suggestion
codecov = { repository = "wasmx/fizzy", service = "github" }
```

File: bindings/rust/Cargo.toml [hunk 4]
```diff
@@ -15,3 +15,8 @@ edition = "2018"
 [build-dependencies]
 bindgen = "0.54.0"
 cmake = "0.1"
+
+[badges]
+circle-ci = { repository = "https://github.com/wasmx/fizzy" }
```
Comment (axic) [inline bindings/rust/Cargo.toml:20]: Not sure if this should be:
```suggestion
circle-ci = { repository = "wasmx/fizzy" }
```

Comment (chfast) [reply bindings/rust/Cargo.toml:20]: I don't know either.

**Comment by axic** [comment]
Merged in #589 to make @chfast happy.

**Pull request merged**